ds_judgements_public_ui/templates/includes/document_text_download_options.html

```diff
@@ -0,0 +1,21 @@
+{% load i18n %}
+<div id="download-options" class="document-download-options">
+  <h2 class="document-download-options__header">{% translate "document.downloadoptions.title" %}</h2>
+  <div class="document-download-options__options-list">
+    <div class="document-download-options__download-option">
+      <h3>
+        <a href="{{ context.pdf_uri }}">
+          {% translate "document.downloadoptions.pdf.cta" %}
```

Comment:
no longer using the conditional 

`{% if context.document_type == 'press_summary' %}` to determine whether to show press_summary or judgment, but instead always show "document" in the download static text. Unless others feel strongly against that. It keeps things simpler this way.

Reply:
Happy with this. We also have `{{ document.document_noun }}` available, but then we need to muck about with string inclusion in translations.